src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -34493,6 +34493,32 @@ public class C
 ");
             Assert.Null(model.GetOperation(node3).Parent);
         }
+
+        [Fact]
+        public void OutVarInConstructorUsedInInitializer()
```

Comment:
Do we have a similar test for collection initializers?

Reply:
Added OutVarInConstructorUsedInCollectionInitializer